### DISCUSSION TITLE
Can't push/commit my local files to GitHub.

### DISCUSSION THREAD

**Discussion by jjgitit** [NvChad/NvChad#2345] [Q&A] [answered]
I've tried at various locations but I don't think any git command is working for my NvChad. When I hit 'space-c-m', it displays the following message. Any help would be greatly appreciated! Thanks.
`Error executing Lua callback: .../share/nvim/lazy/lazy.nvim/lua/lazy/core/handler/cmd.lua:45: Vim:Error executing Lua callback: ...nvim/lazy/telescope.nvim/lua/telescope/
builtin/__git.lua:431: attempt to call field 'tbl_isarray' (a nil value)
stack traceback:
        ...nvim/lazy/telescope.nvim/lua/telescope/builtin/__git.lua:431: in function 'try_worktrees'
        ...nvim/lazy/telescope.nvim/lua/telescope/builtin/__git.lua:477: in function 'set_opts_cwd'
        ...nvim/lazy/telescope.nvim/lua/telescope/builtin/__git.lua:493: in function 'v'
        .../nvim/lazy/telescope.nvim/lua/telescope/builtin/init.lua:584: in function <.../nvim/lazy/telescope.nvim/lua/telescope/builtin/init.lua:543>
        ...share/nvim/lazy/telescope.nvim/lua/telescope/command.lua:188: in function 'run_command'
        ...share/nvim/lazy/telescope.nvim/lua/telescope/command.lua:253: in function 'load_command'
        ...ocal/share/nvim/lazy/telescope.nvim/plugin/telescope.lua:108: in function <...ocal/share/nvim/lazy/telescope.nvim/plugin/telescope.lua:107>
        [C]: in function 'cmd'
        .../share/nvim/lazy/lazy.nvim/lua/lazy/core/handler/cmd.lua:45: in function <.../share/nvim/lazy/lazy.nvim/lua/lazy/core/handler/cmd.lua:16>
stack traceback:
        [C]: in function 'cmd'
        .../share/nvim/lazy/lazy.nvim/lua/lazy/core/handler/cmd.lua:45: in function <.../share/nvim/lazy/lazy.nvim/lua/lazy/core/handler/cmd.lua:16>`

**Comment by siduck** [comment, marked as answer]
is your cwd a git dir? i doubt
